ext/shmop/shmop.stub.php

```diff
@@ -2,20 +2,17 @@
 
 /** @generate-function-entries */
 
-/** @return resource|false */
-function shmop_open(int $key, string $flags, int $mode, int $size) {}
+final class Shmop {}
```

Comment:
What do you think about using a class name like `SharedMemoryBlock`? For now, the chosen class name is mostly opaque, but if we end up adding an OO API, naming would play a more important role.

Reply:
I'm not against a more descriptive name, but `Shmop` has the advantage that it's short and also unlikely to collide.